lib/MusicBrainz/Server/Data/Release.pm

```diff
@@ -1450,19 +1450,6 @@ sub load_meta
         $obj->amazon_store($row->{amazon_store}) if defined $row->{amazon_store};
```

Comment:
We also don't seem to be using this for anything, what am I missing?

Reply:
Nothing that I can see.  Removed it.